cmd/operator-opamp-bridge/agent/testdata/agent.yaml

```diff
@@ -0,0 +1,17 @@
+endpoint: ws://127.0.0.1:4320/v1/opamp
+protocol: wss
+capabilities:
+  - Unspecified
```

Comment:
Nit: "Unspecified" is not supposed to be specifiable by the user and it is a 0 value which doesn't set any capability bit, so it is essentially a no-op even if the user does  specify it.